PropMaker_KeyBlade/code.py

```diff
@@ -1,3 +1,14 @@
+"""
+PyPortal based alarm clock.
+Adafruit invests time and resources providing this open source code.
+Please support Adafruit and open source hardware by purchasing
+products from Adafruit!
+Written by Dave Astels for Adafruit Industries
```

Comment:
@djecken You must have copied and pasted this - it's titled "PyPortal alarm clock" and is attributed to Dave Astels.